crates/ruff_python_formatter/src/statement/stmt_global.rs

```diff
@@ -1,12 +1,18 @@
-use crate::{not_yet_implemented, FormatNodeRule, PyFormatter};
-use ruff_formatter::{write, Buffer, FormatResult};
+use ruff_formatter::{format_args, write, Buffer, FormatResult};
 use ruff_python_ast::StmtGlobal;
 
+use crate::prelude::*;
+use crate::{FormatNodeRule, FormattedIterExt, PyFormatter};
```

Comment:
```suggestion
use crate::{FormatNodeRule};
```

I believe these two should not be necessary if you import `prelude`